agent/bench-scripts/pbench-fio

```diff
@@ -742,6 +759,6 @@ pbench-metadata-log --group=$tool_group --dir=$benchmark_run_dir end
 echo "$script_path/postprocess/generate-benchmark-summary \"$benchmark\" \"$orig_cmd\" \"$benchmark_run_dir\"" >"$benchmark_run_dir/generate-benchmark-summary.cmd"
 chmod +x "$benchmark_run_dir/generate-benchmark-summary.cmd"
 $script_path/postprocess/generate-benchmark-summary "$benchmark" "$orig_cmd" "$benchmark_run_dir"
-pbench-collect-sysinfo --group=$tool_group --dir=$benchmark_run_dir end
+pbench-collect-sysinfo --group=$tool_group --dir=$benchmark_run_dir  --sysinfo=$sysinfo end
```

Comment:
You need to specify --sysinfo on the *other* pbench-collect-sysinfo call as well: the one at the beginning of the run.